clang/docs/checkedc/3C/CONTRIBUTING.md

```diff
@@ -0,0 +1,40 @@
+# Contributing to 3C
+
+Issues and pull requests related to 3C should be submitted to [CCI's repository](https://github.com/correctcomputation/checkedc-clang), not [Microsoft's Checked C repository](https://github.com/microsoft/checkedc-clang), except as stated below.
+
+## Issues
+
+We do not yet have any specific policies about issues beyond the obvious: please try to provide enough information that we can reproduce the bug you are reporting or understand the specific enhancement you are proposing.  Feel free to go ahead and file issues, even at the current early stage of development, to help inform us of what users want; if we find that we're deluged with issues we're unprepared to address, we'll establish a different policy.  We strive to promptly fix any issues that block basic usage of 3C in our [supported environments](INSTALL.md#supported-environments), but beyond that, we have our own plans that are rapidly evolving and you should not expect that issues will be resolved in any given timeframe.
```

Comment:
I think this can be shortened a bit. Proposed rewrite:

Feel free to file issues against 3C, even during this early stage in its development; filed issues will help inform us of what users want. Please try to provide enough information that we can reproduce the bug you are reporting or understand the specific enhancement you are proposing. We strive to promptly fix any issues that block basic usage of 3C in our [supported environments](INSTALL.md#supported-environments).

Reply:
Sure, I'm not going to nitpick this right now.